ui/pages/confirmations/utils/confirm.ts

```diff
@@ -47,3 +48,18 @@ export const parseSanitizeTypedDataMessage = (dataToParse: string) => {
 
 export const isSIWESignatureRequest = (request: SignatureRequestType) =>
   request.msgParams?.siwe?.isSIWEMessage;
+
+export const isPermitSignatureRequest = (request: SignatureRequestType) => {
+  if (
+    !request ||
+    !isSignatureTransactionType(request) ||
+    request.type !== 'eth_signTypedData' ||
+    request.msgParams?.version?.toLowerCase() === 'v1'
```

Comment:
Do we have a constant or enum we can use for the versions?

Reply:
Done 👍